etc/zones/db.ocf

```diff
@@ -202,147 +262,220 @@ radiation	IN AAAA	2607:f140:8801::1:32
 
 ; supernova (server)
+_acme-challenge.supernova	IN CNAME	supernova.letsencrypt.ocf.io.
 supernova	IN A	169.229.226.36
 supernova	IN AAAA	2607:f140:8801::1:36
+_acme-challenge.admin	IN CNAME	admin.letsencrypt.ocf.io.
 admin	IN CNAME	supernova
+_acme-challenge.create	IN CNAME	create.letsencrypt.ocf.io.
 create	IN CNAME	supernova
 
 ; dev-death (server)
+_acme-challenge.dev-death	IN CNAME	dev-death.letsencrypt.ocf.io.
 dev-death	IN A	169.229.226.37
 dev-death	IN AAAA	2607:f140:8801::1:37
+_acme-challenge.dev-accounts	IN CNAME	dev-accounts.letsencrypt.ocf.io.
 dev-accounts	IN CNAME	dev-death
+_acme-challenge.dev-hello	IN CNAME	dev-hello.letsencrypt.ocf.io.
 dev-hello	IN CNAME	dev-death
+_acme-challenge.dev-ocf-io	IN CNAME	dev-ocf-io.letsencrypt.ocf.io.
 dev-ocf-io	IN CNAME	dev-death
+_acme-challenge.dev-wiki	IN CNAME	dev-wiki.letsencrypt.ocf.io.
 dev-wiki	IN CNAME	dev-death
+_acme-challenge.dev-www	IN CNAME	dev-www.letsencrypt.ocf.io.
 dev-www	IN CNAME	dev-death
+_acme-challenge.dev-dev-vhost	IN CNAME	dev-dev-vhost.letsencrypt.ocf.io.
 dev-dev-vhost	IN A	169.229.226.37
 dev-dev-vhost	IN AAAA	2607:f140:8801::1:37
+_acme-challenge.dev-dev-vhost-alias	IN CNAME	dev-dev-vhost-alias.letsencrypt.ocf.io.
 dev-dev-vhost-alias	IN A	169.229.226.37
 dev-dev-vhost-alias	IN AAAA	2607:f140:8801::1:37
 
 ; dev-firestorm (server)
+_acme-challenge.dev-firestorm	IN CNAME	dev-firestorm.letsencrypt.ocf.io.
 dev-firestorm	IN A	169.229.226.38
 dev-firestorm	IN AAAA	2607:f140:8801::1:38
+_acme-challenge.dev-kerberos	IN CNAME	dev-kerberos.letsencrypt.ocf.io.
 dev-kerberos	IN CNAME	dev-firestorm
+_acme-challenge.dev-ldap	IN CNAME	dev-ldap.letsencrypt.ocf.io.
 dev-ldap	IN CNAME	dev-firestorm
 
 ; dev-anthrax (server)
+_acme-challenge.dev-anthrax	IN CNAME	dev-anthrax.letsencrypt.ocf.io.
 dev-anthrax	IN A	169.229.226.39
 dev-anthrax	IN AAAA	2607:f140:8801::1:39
 
 ; dev-pestilence (server)
+_acme-challenge.dev-pestilence	IN CNAME	dev-pestilence.letsencrypt.ocf.io.
 dev-pestilence	IN A	169.229.226.40
 dev-pestilence	IN AAAA	2607:f140:8801::1:40
+_acme-challenge.dev-dhcp	IN CNAME	dev-dhcp.letsencrypt.ocf.io.
 dev-dhcp	IN CNAME	dev-pestilence
+_acme-challenge.dev-rancid	IN CNAME	dev-rancid.letsencrypt.ocf.io.
 dev-rancid	IN CNAME	dev-pestilence
+_acme-challenge.dev-ns	IN CNAME	dev-ns.letsencrypt.ocf.io.
 dev-ns	IN A	169.229.226.40
 dev-ns	IN AAAA	2607:f140:8801::1:40
 
 ; gridlock (server)
+_acme-challenge.gridlock	IN CNAME	gridlock.letsencrypt.ocf.io.
 gridlock	IN A	169.229.226.41
 gridlock	IN AAAA	2607:f140:8801::1:41
+_acme-challenge.broker	IN CNAME	broker.letsencrypt.ocf.io.
 broker	IN CNAME	gridlock
 
 ; dev-werewolves (server)
+_acme-challenge.dev-werewolves	IN CNAME	dev-werewolves.letsencrypt.ocf.io.
 dev-werewolves	IN A	169.229.226.42
 dev-werewolves	IN AAAA	2607:f140:8801::1:42
+_acme-challenge.dev-apphost	IN CNAME	dev-apphost.letsencrypt.ocf.io.
 *.dev-apphost	IN CNAME	dev-werewolves
+_acme-challenge.dev-apphost	IN CNAME	dev-apphost.letsencrypt.ocf.io.
```

Comment:
There will be some duplicate records doing it this way (removing the wildcard from the beginning), but bind seems to handle it fine. Not sure what would happen if they conflict, but probably would get two results back or something? Anyway, none of our records currently have conflicts where the wildcard domain points somewhere different from the domain without the wildcard, so that's fine for now.